CppCoreGuidelines.md

```diff
@@ -9842,31 +9846,11 @@ This makes the function's reseating explicit.
 
 ##### Example, good
 
-    void share(shared_ptr<widget>);            // share -- "will" retain refcount
-
-    void reseat(shared_ptr<widget>&);          // "might" reseat ptr
-
-    void may_share(const shared_ptr<widget>&); // "might" retain refcount
-
-##### Enforcement
```

Comment:
why delete R.35 enforcement?

Reply:
This was R.36's enforcement.  R.35 and R.36 have the same enforcement and the diff just showed it this way.